internal/infrastructure/kubernetes/infra.go

```diff
@@ -13,6 +14,11 @@ import (
 	"github.com/envoyproxy/gateway/internal/ir"
 )
 
+const (
+	// envoyGatewayNamespace is the namespace where envoy-gateway is running.
+	envoyGatewayNamespace = "envoy-gateway-system"
```

Comment:
How about making this const exported? I think envoy-gateway-system will be used not only here in the future. It is like the root namespace of EG.

Reply:
* currently the usage of `envoy-gateway-system` is only in YAML.
* I prefer making the const private now and changing it to visible outside the package once such a case exists